site/app/templates/calendar/Calendar.twig

```diff
@@ -62,10 +119,31 @@
                     <i class="fas fa-plus"></i> New Item
                 </button>
             {% endif %}
-
                 <button type="button" onclick="openOptionsModal()" class="btn btn-primary">
                     <i class="fas fa-sliders"></i> Options
                 </button>
+                <select id="calendar-year-type-edit" name="type" onchange="changeYear(this);">
+                    {% set startYear = curr_year - 20 %}
```

Comment:
Is 20 years before and after today what we want here, or should we instead run something like 2014 (Submitty's release) - present?

Reply:
good idea I will change that